pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/db/dto/CompositeQuestionDto.java

```diff
@@ -16,20 +16,23 @@ public final class CompositeQuestionDto extends QuestionDto implements Serializa
     private final Long addButtonTemplateId;
     private final Long additionalItemTemplateId;
     private final OrientationType childOrientation;
+    private final String tabularSeparator;
 
     @JdbiConstructor
     public CompositeQuestionDto(@Nested QuestionDto questionDto,
                                 @ColumnName("allow_multiple") boolean allowMultiple,
                                 @ColumnName("unwrap_on_export") boolean unwrapOnExport,
                                 @ColumnName("add_button_template_id") Long addButtonTemplateId,
                                 @ColumnName("additional_item_template_id") Long additionalItemTemplateId,
-                                @ColumnName("child_orientation") OrientationType childOrientation) {
+                                @ColumnName("child_orientation") OrientationType childOrientation,
+                                @ColumnName("tabularSeparator") String tabularSeparator) {
```

Comment:
Is this column name correct? The other columns use snake case instead of camel case.